[2023-04-12 18:15:44,166] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v1.extract_data_task manual__2023-04-12T18:15:33.941608+00:00 [queued]>
[2023-04-12 18:15:44,208] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v1.extract_data_task manual__2023-04-12T18:15:33.941608+00:00 [queued]>
[2023-04-12 18:15:44,210] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-04-12 18:15:44,212] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-04-12 18:15:44,214] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-04-12 18:15:44,261] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): extract_data_task> on 2023-04-12 18:15:33.941608+00:00
[2023-04-12 18:15:44,273] {standard_task_runner.py:52} INFO - Started process 635 to run task
[2023-04-12 18:15:44,282] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v1', 'extract_data_task', 'manual__2023-04-12T18:15:33.941608+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/sp_500_data_processing_dag.py', '--cfg-path', '/tmp/tmpi60ck22x', '--error-file', '/tmp/tmpnvjeyovo']
[2023-04-12 18:15:44,285] {standard_task_runner.py:77} INFO - Job 209: Subtask extract_data_task
[2023-04-12 18:15:44,418] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v1.extract_data_task manual__2023-04-12T18:15:33.941608+00:00 [running]> on host 7b33c6d8ff37
[2023-04-12 18:15:44,572] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v1
AIRFLOW_CTX_TASK_ID=extract_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-04-12T18:15:33.941608+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-12T18:15:33.941608+00:00
[2023-04-12 18:15:54,799] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas_datareader/tiingo.py:234: FutureWarning: In a future version of pandas all arguments of concat except for the argument 'objs' will be keyword-only
  return pd.concat(dfs, self._concat_axis)

[2023-04-12 18:16:26,304] {local_task_job.py:212} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-04-12 18:16:26,342] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 635. PIDs of all processes in the group: [635]
[2023-04-12 18:16:26,356] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 635
[2023-04-12 18:16:26,361] {taskinstance.py:1408} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-04-12 18:16:26,368] {logging_mixin.py:109} INFO - Error while extracting data for AEE: Task received SIGTERM signal
[2023-04-12 18:17:11,631] {logging_mixin.py:109} INFO - Error while extracting data for BRK.B: Unable to read URL: https://api.tiingo.com/tiingo/daily/BRK.B/prices?startDate=2018-04-13&endDate=2023-04-12&format=json
Response Text:
b'{"detail":"Error: Ticker \'BRK.B\' not found"}'
[2023-04-12 18:17:26,388] {process_utils.py:138} WARNING - process psutil.Process(pid=635, name='airflow task ru', status='running', started='18:15:43') did not respond to SIGTERM. Trying SIGKILL
[2023-04-12 18:17:26,407] {process_utils.py:75} INFO - Sending the signal Signals.SIGKILL to group 635
[2023-04-12 18:17:26,508] {process_utils.py:70} INFO - Process psutil.Process(pid=635, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='18:15:43') (635) terminated with exit code Negsignal.SIGKILL
[2023-04-12 18:17:26,514] {standard_task_runner.py:135} ERROR - Job 209 was killed before it finished (likely due to running out of memory)
